vars/warnAboutOldTfAzureProvider.groovy

```diff
@@ -0,0 +1,18 @@
+import uk.gov.hmcts.pipeline.deprecation.WarningCollector
+
+def call() {
+
+  writeFile file: 'warn-about-old-tf-azure-provider.sh', text: libraryResource('uk/gov/hmcts/helm/warn-about-old-tf-azure-provider.sh')
+
+  try {
+    sh """
+    chmod +x warn-about-old-tf-azure-provider.sh
+    ./warn-about-old-tf-azure-provider.sh
+    """
+  } catch(ignored) {
+    WarningCollector.addPipelineWarning("updated_azurerm_provider", "Please upgrade azurerm to the lastest 2.x version. This will allow us to upgrade our KeyVault module to the lateat Terraform and Azurerm versions", new Date().parse("dd.MM.yyyy", "22.09.2020"))
```

Comment:
discussed with darren today, we may want to make this a bit longer, in case there's any teams like IDAM who this is more work for.

How about this?
```suggestion
    WarningCollector.addPipelineWarning("updated_azurerm_provider", "Please upgrade azurerm to the lastest 2.x version. This will allow us to upgrade our KeyVault module to the lateat Terraform and Azurerm versions", new Date().parse("dd.MM.yyyy", "06.10.2020"))
```

Reply:
Yeah, I was thinking that might be too close a deadline